### ISSUE TITLE
[iPad] Screen shows in portrait mode when the menu is shown

### ISSUE THREAD

**Issue by MaartenWeber** [myflashlab/PDF-ANE#1]
When I open a pdf and the menu is shown on my iPad 4, the app changes to a portrait mode and is blurred. When I close the menu, the app returns to landscape mode.

Since the app is landscape mode only, the app shouldn't switch to portrait mode.

**Comment by myflashlab** [comment]
We tested on iPad and we were not able to duplicate the problem you mentioned as you see in the screenshot below. But we accidently noticed another minor bug and fixed that. please update yourself with V2.0.1 anyway.

![img_2649](https://cloud.githubusercontent.com/assets/1692948/12761689/37d693e2-ca01-11e5-9f74-a48d16e31bc5.JPG)

**Comment by MaartenWeber** [comment]
Thanks, I'll download the update and try again. I'm not at the office right now, so I can't test it immediately. As soon as I've tested it I'll create a screenshot.

I'll also test it on other iDevices. 

Oh, by the way. I forgot to give some more info (stupid me) on the versions:

iPad 4
iOS 8.3
Air version 15

Haven't updated air in a while now come to think of it... might that be the cause of it?

**Comment by myflashlab** [comment]
Oh, yes! That's VERY probable. Few months ago we reported that orientation
bug in iOS to Adobe. You should upgrade to Air 20. That should fix it i
guess.

On Tuesday, February 2, 2016, MaartenWeber <notifications@github.com> wrote:

> Thanks, I'll download the update and try again. I'm not at the office
> right now, so I can't test it immediately. As soon as I've tested it I'll
> create a screenshot.
>
> I'll also test it on other iDevices.
>
> Oh, by the way. I forgot to give some more info (stupid me) on the
> versions:
>
> iPad 4
> iOS 8.3
> Air version 15
>
> Haven't updated air in a while now come to think of it... might that be
> the cause of it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/myflashlab/PDF-ANE/issues/1#issuecomment-178787674>.
>


-- 
Sent from iPhone

**Comment by myflashlab** [comment]
@MaartenWeber Please let us know if the problem is solved?

**Comment by MaartenWeber** [comment]
Oh hehe, sorry. Totally forgot to reply! I've updated my Air to version 20 and now it doesn't flip anymore on iOS! :). I do briefly see the app change from it's place, as if a snapshot is being taken and place on top of it. Don't really know how to describe it better though. It happens pretty fast.

Concerning the android version (which is a bit of a different thing) I noticed a prompt opening twice on my phone. As if the call is being called double. Though the first prompt offers more programs to open the pdf than the second one. Which is showing when I return to the app. Don't know if it was already the case in version 2.0 of the ANE. I'd have to check that. I still have an app built with that version lying around I think.

Those are my findings. Both aren't game breaking for me at the moment.
